### PR TITLE
Merge the goval polygott with the polygott polygott

### PR DESCRIPTION
**Pull request by lhchavez**
Having two copies of the same thing is confusing ^^;; Although this
requires a rebuild, I'm planning on making that faster.

A few things had to be fixed in this change:

* Swift's key had expired. The latest and greatest was added from
  https://swift.org/download/#using-downloads
* One of pyxel's test was missing a newline.
* The assembly test was returning 1 and "failing" the test.
* There was no clear indication whether a test had failed because the
  project run failed or because the diff failed. It was split in two
  steps to be able to better discern what happened.

And one more thing:

* Added `DOCKER_BUILDKIT=1` because I got fed up with the build times.
  2ksecond builds?!

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,24 +4,24 @@
 # Default task:
 .PHONY: image
 image: ## Build Docker image with all languages
-	docker build -t polygott:latest .
+	DOCKER_BUILDKIT=1 docker build -t polygott:latest .
```
Comment (dan-stowell) [inline Makefile:7]: Nit: can you add `--progress=plain` to these lines? Yay buildkit!

**Review by dan-stowell — APPROVED** [review]
Hear hear!

**Comment by dan-stowell** [comment]
Now that I think of it, is there way to make `polygott-x11-vnc` a no-op, just to keep repls that depend on it running?

File: Makefile (hunk 1 above)
Comment (lhchavez) [reply Makefile]: i can!

**Comment by lhchavez** [comment]
> Now that I think of it, is there way to make `polygott-x11-vnc` a no-op, just to keep repls that depend on it running?

let's do it, in the name of backwards-compatibility.

**Pull request merged**